Specs/DateTime/English/DatePeriodParser.json

```diff
@@ -3187,4 +3187,108 @@
     }
   ]
 }
+,
+{
+  "Input": "It happened in the past 2 decades.",
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "the past 2 decades",
+      "Type": "daterange",
+      "Value": {
+        "Timex": "(1990-01-01,2009-12-31,P20Y)",
```

Comment:
See the discussion on your other PR for this. range and timex should be non-inclusive at the end.
Same for resolution.